move_only_io_iterator/P1862.tex

```diff
@@ -2167,12 +2161,12 @@ \section{Wording}
 		{"Babbage"sv, 1791},
 		{"Hamilton"sv, 1936}
 	};
-	
+
 	auto names = historical_figures | views::elements<0>;
 	for (auto&& name : names) {
 		cout << name << ' ';          // prints \tcode{Babbage Hamilton Lovelace Turing }
```

Comment:
Huh - any idea what this crazy red markup means to GitHub?

Reply:
nope

Reply:
The interwebs suggest that this dark red markup indicates that GitHub believes there's a syntax error in the file. I verified that these three characters are exactly `' '`, so that's not the case here. Oh well.